Firebase/android/app/src/main/res/values/users_dimens.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <dimen name="user_image_size">36dp</dimen>
```

Comment:
sometimes there is a blank line at the top, sometimes at the bottom, sometimes both/neither :sob: